docs/guides/openshock-first-setup.md

```diff
@@ -4,6 +4,10 @@
     **Don't wear the shocker somewhere near your neck or your heart.**  
     Check out [Safety](../safety/safety-rules.md) for more information.
 
+!!! danger "Safety Warning" 
+    **Do not touch the pins of the shocker with both hands at the same time.** 
+    The electricity will flow through your heart. 
```

Comment:
I would rather try to warn the user to try to no touch with both hands when its powered on because it *could* send electricity trough the heart.